tests/observability/metrics/utils.py

```diff
@@ -1153,3 +1154,19 @@ def compare_metric_file_system_values_with_vm_file_system_values(
             f"Result from metric for the mountpoint: {mount_point}: {metric_value}"
         )
         raise
+
+
+def compare_metric_labels_value_with_expected_results(
+    prometheus: Prometheus, metric_name: str, expected_labels_and_values: dict[str, str]
```

Comment:
This are only storage class labels, then use more meaningful name please. expected_storage_class_labels_and_values